sdk/postgresql/armpostgresql/go_mod_tidy_hack.go

```diff
@@ -9,4 +9,4 @@
 package armpostgresql
 
 // Necessary for safely adding multi-module repo. See: https://github.com/golang/go/wiki/Modules#is-it-possible-to-add-a-module-to-a-multi-module-repository
-import _ "github.com/Azure/azure-sdk-for-go"
\ No newline at end of file
+import _ "github.com/Azure/azure-sdk-for-go"
```

Comment:
There needs to be a new line here or the format check step of pipeline verification will fail